src/components/Articles.jsx

```diff
@@ -22,12 +22,14 @@ const Articles = () => {
 				) : (
 					<>
 						{articles.map(({ article_id, topic, title, author, created_at, votes, comment_count, img_url }) => (
-							<div key={article_id}>
-								<h3>{topic}</h3>
+							<div className='articles-card-div' key={article_id}>
```

Comment:
This mapped data to html elements could have been a react component to improve readability. Will leave up to you to decide whether to implement.